dev-lang/lua/patches/lua-5.4.7.gcc2.patchset

```diff
@@ -22,3 +22,27 @@ index f2a74ab..853680f 100644
 -- 
 2.45.1
 
+
+From aaf3345e5ea94492b0677ca3585642d99cfde702 Mon Sep 17 00:00:00 2001
+From: Schrijvers Luc <begasus@gmail.com>
+Date: Fri, 5 Jul 2024 13:16:05 +0200
+Subject: Fix for 32bit LUA_NOBUILDIN CFLAG doesn't seem to respect this on
```

Comment:
This was mentioned in: https://github.com/haikuports/haikuports/pull/6707#issuecomment-1066107200

Seems that didn't really fix the problem, see: https://github.com/haikuports/haikuports/issues/8874#issuecomment-2210628451